### PR TITLE
Update release notes with #16739

### PR DESCRIPTION
**Pull request by ares**
Note sure where to put it exactly, normally there's just list of issues. Please let me know if I should move it somewhere. Also please do a grammar check.

### CODE REVIEW

**Comment by domcleal** [comment]
Could you move it underneath "Upgrade warnings" at the bottom of the file?

File: _includes/manuals/1.14/1.2_release_notes.md [hunk 1]
```diff
@@ -8,6 +8,8 @@
 
 #### Authentication and Authorization
 
+* Parameter permissions were unified under Parameter resource type, all filters for Global Parameters are automatically migrated to new permission names. If user was granted one of view_params, edit_params, destroy_params, create_params, they now apply to Global Parameters too. Common Parameter permissions view_globals, edit_globals, create_globals, destroy_globals were removed. The same functionality can be achieved by specifying filter condition like `type = CommonParameter`. For more information, see ([#16739](http://projects.theforeman.org/issues/16739))
```
Comment (domcleal) [inline _includes/manuals/1.14/1.2_release_notes.md]: 1. unified under _the_ Parameter resource type
1. Global Parameters should probably be Common Parameters, as that's how it's displayed in the filters (or add "Global Parameters" in brackets).

**Review by domcleal — CHANGES_REQUESTED** [review]

**Comment by ares** [comment]
Updated and moved, thanks for quick looking.

**Comment by domcleal** [comment]
Thanks @ares.

**Pull request merged**